rollup.config.js

```diff
@@ -0,0 +1,85 @@
+import babel from 'rollup-plugin-babel'
+import commonjs from '@rollup/plugin-commonjs'
+import replace from '@rollup/plugin-replace'
+import resolve from '@rollup/plugin-node-resolve'
+import { terser } from 'rollup-plugin-terser'
+import pkg from './package.json'
+
+const { peerDependencies } = pkg
+const external = Object.keys(peerDependencies)
```

Comment:
Can you add some documentation comments to this file to explain why we do things like this?

Reply:
Yep. I will do it.